bootstrap/shoppingmall_review_seed.py

```
@@ -0,0 +1,144 @@
+        rating = random.choice([1.0, 1.0, 2.0, 2.0])
+        template = random.choice(NEGATIVE_TEMPLATES)
+    else:
+        rating = 3.0
+        template = random.choice(NEUTRAL_TEMPLATES)
+    return {
+        "id": idx,
+        "product_id": pid,
+        "user_id": random.choice(user_ids),
+        "rating": rating,
+        "content": template.format(product=pname),
+        "created_at": datetime.utcnow() - timedelta(days=random.randint(0, 90)),
```

Ruff (0.15.10)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)